content/en/docs/reference/scheduling/config.md

```diff
@@ -199,6 +202,9 @@ that are not enabled by default:
   configured labels. This plugin also favors spreading the Pods belonging to a
   Service across nodes.
   Extension points: `PreFilter`, `Filter`, `Score`.
+- `NodePreferAvoidPods`: Prioritizes nodes according to the node annotation
+  `scheduler.alpha.kubernetes.io/preferAvoidPods`.
+  Extension points: `Score`.
   
```

Comment:
(not related with this line)

Update L261 to link to v1beta2.

Reply:
Done